scope.py

```diff
@@ -529,10 +532,16 @@ def train(
         from wandb.keras import WandbCallback
 
         from scope.nn import DNN
+        from scope.xgb import XGB
         from scope.utils import Dataset
 
         train_config = self.config["training"]["classes"][tag]
 
+        if algorithm in ['DNN', 'NN', 'dnn', 'nn']:
+            algorithm = 'dnn'
+        elif algorithm in ['XGB', 'xgb', 'XGBoost', 'xgboost', 'XGBOOST']:
+            algorithm = 'xgb'
```

Comment:
We should raise an error if we don't find one?

Reply:
Agreed, will implement in the next push!